fs/react-next-base
# File upload 52

# Summary

#### A brief description of the pull request.

#### Resolves: #52 

#### Screenshots in case of UI changes
![image](https://user-images.githubusercontent.com/51324247/94153255-387c5800-fe85-11ea-8f8f-f9514a0412a3.png)
![image](https://user-images.githubusercontent.com/51324247/94153517-8d1fd300-fe85-11ea-8b8c-f19347424a3c.png)

# Test plan

#### List of steps to manually test introduced functionality:

- Pull `file-upload-52` branch
- Go to [Profile Page](http://localhost:8080/profile)
- Update profile with updating avatar
- The profile image should be updated

#### What do need to pay more attention to, some implicit changes that influence the app:

- ...

# Review notes

#### While reviewing pull-request (especially when it's your pull-request), please make sure that:

- you named PR close to issue name and linked that issue
- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes
